Sources/SwiftDate/DateInRegionFormatter.swift

```diff
@@ -327,6 +330,65 @@ public class DateInRegionFormatter {
 		localized_str = String(format: localized_str, arguments: arguments)
 		return localized_str
 	}
+    
+    /// Return true if date for russian locale, number are plural and localized strings has right translate for this unit
+    ///
+    /// - parameter unit:     unit of calendar component
+    /// - parameter value:    the value associated with the unit
+    ///
+    /// - returns: true or false
+    private func isRussianSpecifyCase(unit: Calendar.Component, withValue value: Int) -> Bool {
+        if locale?.identifier.hasPrefix("ru") == false {
```

Comment:
- Use `guard` statements instead of `== false`.
- `locale?.identifier.hasPrefix("ru") == false` evaluates to false if `locale` is `nil`.